indico/modules/events/controllers/creation.py

```diff
@@ -123,3 +137,44 @@ def _process(self):
                                 event_type=self.event_type.name, single_category=self.single_category,
                                 check_room_availability=check_room_availability,
                                 rb_excluded_categories=rb_excluded_categories)
+
+class RHCreateEventUUID(RH):
```

Comment:
I don't think we need a new class here. Let's just pass the UUID as a query string argument to the one we already have and populate the form with the details from cache in that case instead of the usual defaults.

Reply:
I can't recall why this was added (it's not used). Removed